src/index.css

```diff
@@ -7,6 +7,33 @@
 }
 
 
+.jp-FileBrowser-breadcrumbs {
+  padding: 5px;
```

Comment:
margin and padding are subjective quanties that a theme will likely want to change. If this were a Phosphor repo, I would move these to example CSS. However, this is not a Phosphor repo and we can probably be more opinionated. We should probably setup a discussion with Brian and others to decide how much styling/theming we want to provide in the core.